### ISSUE TITLE
Crashing upon slideshow start

### ISSUE THREAD

**Issue by omega7pro** [avluis/Hentoid#1162]
The Issue: Simply put, every time I try to use the slideshow function the app crashes/restarts.

**Hentoid Version**: v1.19.6 (728)

**Device/Android Version**: Galaxy Tab A (8.4", 2020)/SM-T307U

**Issue details / Repro steps**: Open up anything and hit the "Slideshow" button. It should crash/restart right when the slideshow time begins.

**Stack trace / LogCat**: 
```latest-crash log : begin
Hentoid ver: 1.19.6 (728)
API: 30
Device: SM-T307U
Start : 2024-05-20T21:44:06.058Z
End : 2024-05-20T21:44:06.059Z (00:00)
-----
f != [Ljava.lang.Object;
java.util.IllegalFormatConversionException: f != [Ljava.lang.Object;
	at java.util.Formatter$FormatSpecifier.failConversion(Formatter.java:4403)
	at java.util.Formatter$FormatSpecifier.printFloat(Formatter.java:2898)
	at java.util.Formatter$FormatSpecifier.print(Formatter.java:2845)
	at java.util.Formatter.format(Formatter.java:2524)
	at java.util.Formatter.format(Formatter.java:2459)
	at java.lang.String.format(String.java:2911)
	at android.content.res.Resources.getString(Resources.java:630)
	at android.content.Context.getString(Context.java:674)
	at me.devsaki.hentoid.util.ToastHelperKt.toast(SourceFile:0)
	at me.devsaki.hentoid.util.ToastHelperKt.toastShort(SourceFile:0)
	at me.devsaki.hentoid.util.ToastHelperKt.toast(SourceFile:0)
	at me.devsaki.hentoid.fragments.reader.ReaderPagerFragment.startSlideshow(Unknown Source:108)
	at me.devsaki.hentoid.fragments.reader.ReaderPagerFragment.onSlideShowSliderChosen(Unknown Source:42)
	at me.devsaki.hentoid.fragments.reader.ReaderPagerFragment.access$onSlideShowSliderChosen(Unknown Source:0)
	at me.devsaki.hentoid.fragments.reader.ReaderPagerFragment$onCreate$2.invoke(SourceFile:0)
	at me.devsaki.hentoid.fragments.reader.ReaderPagerFragment$onCreate$2.invoke(SourceFile:0)
	at me.devsaki.hentoid.util.Debouncer$submit$1.invokeSuspend(Unknown Source:49)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTaskKt.resume(Unknown Source:61)
	at kotlinx.coroutines.DispatchedTaskKt.dispatch(Unknown Source:51)
	at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(Unknown Source:7)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl(Unknown Source:37)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$default(Unknown Source:7)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeUndispatched(Unknown Source:28)
	at kotlinx.coroutines.android.HandlerContext$scheduleResumeAfterDelay$$inlined$Runnable$1.run(Unknown Source:6)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:246)
	at android.app.ActivityThread.main(ActivityThread.java:8615)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@a4149e9, Dispatchers.Main.immediate]

latest-crash log : end
```

@AVnetWS/admin-team

**Comment by omega7pro** [comment]
I hope I did the crash log thing correctly

**Comment by RobbWatershed** [comment]
Thanks for your feedback. This is a known issue that will be fixed for the next release.

**Comment by omega7pro** [comment]
Good to know where to look next if I don't see something on the issue board.

**Comment by RobbWatershed** [comment]
Fixed on today's v1.19.7
